### DISCUSSION TITLE
WLAN PI How to Capture WIFI6 OFDMA Data Frames ?

### DISCUSSION THREAD

**Discussion by xiaodaimu710** [WLAN-Pi/feedback#57] [General Inquiries]
## Question 1

WLAN PI M4  intel AX210   MT7921K  How to Capture WIFI6 OFDMA Data Frames ?

The WLAN PI system does not have this directory：sys/kernel/debug/iwlwifi，What to do？thanks

 - AX210:    https://gjermundraaen.com/2019/11/16/wireless-capturing-of-multi-user-ofdma-frames/
 - MT7921K :    https://patchwork.kernel.org/project/linux-wireless/patch/20230505000543.693532-3-greearb@candelatech.com/

## Question 2 

 wlanpi hostapd mode not support 11ax, is it possible that hostapd installs without removing the # before

```bash
#CONFIG_IEEE80211AX=y
```

like

```bash
CONFIG_IEEE80211AX=y
```

Example:

```bash
The following is from the hostapd installation directory: defconfig 
------------------------------------------------------------------------------------
# IEEE 802.11ax HE support
# Note: This is experimental and work in progress. The definitions are still
# subject to change and this should not be expected to interoperate with the
# final IEEE 802.11ax version.
#CONFIG_IEEE80211AX=y
------------------------------------------------------------------------------------
```

thanks**

**Comment by joshschmelzle** [comment]
Hi @xiaodaimu710, for question 1 and mt76, you can try to apply this series.

https://patchwork.kernel.org/project/linux-wireless/list/?series=745125&archive=both&state=

[v2,3/6] wifi: mt76: mt7915: Support setting aid when in monitor mode

**Comment by joshschmelzle** [comment]
Hi @xiaodaimu710, for question 2, we are installing hostapd from bullseye-backports on our latest OS releases and are not compiling from source. You are welcome to uninstall the shipped version and recompile hostapd from source with your target definitions.
